api/config/cdr_versions_local.json

```diff
@@ -3,6 +3,7 @@
     "cdrVersionId": 1,
     "name": "[Removed] Synthetic Dataset v1",
     "dataAccessLevel": 1,
+    "accessTier": "registered",
```

Comment:
The two lines above mean the same thing.  Perhaps the real version of the Controlled Tier will have an int ID that corresponds to the existing enum.  The prototype uses a string in the config file.

Reply:
Some of this is in the design doc IIRC.